nautobot/ipam/forms.py

```diff
@@ -1129,6 +1136,41 @@ class VLANFilterForm(NautobotFilterForm, LocatableModelFilterFormMixin, TenancyF
         required=False,
         null_option="None",
     )
+    prefixes = DynamicModelMultipleChoiceField(
+        queryset=Prefix.objects.all(),
+        to_field_name="vlan_id",
+        required=False,
+        null_option="None",
+    )
+    has_prefixes = forms.NullBooleanField(
+        required=False,
+        label="Has a prefix",
+        widget=StaticSelect2(choices=BOOLEAN_WITH_BLANK_CHOICES),
+    )
+    interfaces_as_untagged = DynamicModelMultipleChoiceField(
+        queryset=Interface.objects.all(),
+        to_field_name="untagged_vlan",
+        required=False,
+        null_option="None",
+    )
+    interfaces_as_tagged = DynamicModelMultipleChoiceField(
+        queryset=Interface.objects.all(),
+        to_field_name="tagged_vlans",
+        required=False,
+        null_option="None",
+    )
+    vminterfaces_as_untagged = DynamicModelMultipleChoiceField(
+        queryset=VMInterface.objects.all(),
+        to_field_name="untagged_vlan",
+        required=False,
+        null_option="None",
+    )
+    vminterfaces_as_tagged = DynamicModelMultipleChoiceField(
+        queryset=VMInterface.objects.all(),
+        to_field_name="tagged_vlans",
+        required=False,
+        null_option="None",
+    )
```

Comment:
Could we abbreviate this and let some of these only be accessible from the dynamic filter form now that it exists?

In 2.0 the basic/simple filterform may even go away entirely.

Reply:
@bryanculver Yes, thanks for offering, I'm short on time the next few weeks

Reply:
No worries! We’ve begun using the “Draft” status to help us sift through PRs that have been reviewed and are awaiting updates from the author(s), even our own.

Never a rush. Whenever you do get updates feel free to mark it as ready to review again.